### PR TITLE
New Signals

### PR DESCRIPTION
**Pull request by bfilar**

### CODE REVIEW

**Review by morriscode — APPROVED** [review]

File: signals/headers/headers_no_recipients_in_org_domain.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
-name: "Headers: No recpients in $org_domains list"
```
Comment (rw-access) [inline signals/headers/headers_no_recipients_in_org_domain.yml:1]: FYI this needs to be changed in both Mantis and Hydra (we were matching the typo before)

File: signals/sender/sender_new_domain_L90D.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+name: "Sender: Domain registered less than 90 days ago"
+type: "query"
+source: |
+  beta.whois(sender.email.domain).days_old < 90
```
Comment (rw-access) [inline signals/sender/sender_new_domain_L90D.yml:4]: Why not just make the signal on `.days_old` instead?

File: signals/headers/headers_no_recipients_in_org_domain.yml (hunk 1 above)
Comment (bfilar) [reply signals/headers/headers_no_recipients_in_org_domain.yml:1]: yep... I'll pick it up during model training, and then PR an updated config in Hydra

File: signals/sender/sender_new_domain_L90D.yml (hunk 2 above)
Comment (bfilar) [reply signals/sender/sender_new_domain_L90D.yml:4]: yes... that would work too :|

**Pull request merged**